content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -299,25 +299,27 @@ section.
 
 To set security constraints on Pods and containers, you use the
 `securityContext` field in the Pod specification. This field gives you
-granular control over what a Pod or individual containers can do. For example:
-
-* Drop specific Linux capabilities to avoid the impact of a CVE.
-* Force all processes in the Pod to run as a non-root user or as a specific
-  user or group ID.
-* Set a specific seccomp profile.
-* Set Windows security options, such as whether containers run as HostProcess.
-
-{{< caution >}}
-You can also use the Pod securityContext to enable
-[_privileged mode_](/docs/concepts/security/linux-kernel-security-constraints/#privileged-containers)
-in Linux containers. Privileged mode overrides many of the other security
-settings in the securityContext. Avoid using this setting unless you can't grant
-the equivalent permissions by using other fields in the securityContext.
-In Kubernetes 1.26 and later, you can run Windows containers in a similarly
-privileged mode by setting the `windowsOptions.hostProcess` flag on the
-security context of the Pod spec. For details and instructions, see
-[Create a Windows HostProcess Pod](/docs/tasks/configure-pod-container/create-hostprocess-pod/).
-{{< /caution >}}
+granular control over what a Pod or individual containers can do.
```

Comment:
Since the PR removes the general overview of what general control can be done. Perhaps it's good to link to the new "advanced-pod-config" page. Something like the below suggestion
```suggestion
granular control over what a Pod or individual containers can do. See [Advanced Pod Configuration](/docs/concepts/workloads/pods/advanced-pod-config/) for more details.
```

Reply:
Yeah, this is a Good suggestion @reylejano . Thanks.